Makefile

```diff
@@ -44,7 +50,7 @@ docker-multi-arch: .info .cp-docker
 # Build rep & deb packages
 #####
 build-linux: .info .prepare-builder .cp-linux
-	docker run -v $(shell pwd)/:/proxy proxy-linux-builder /proxy/pkg/build.sh ${VERSION} ${REVISION}
+	docker run -v $(shell pwd)/:/proxy proxy-linux-builder /proxy/pkg/build.sh ${VERSION} 1
```

Comment:
Put the "1" just for GA, that way we can have multiple noGA version on packagecloud.